### DISCUSSION TITLE
AWS Bedrock Claude 3 Support

### DISCUSSION THREAD

**Discussion by meenonboard** [truera/trulens#1293] [General]
Hi, My team is interesting in using Trulens for Evaluation LLM with AWS Bedrock.
My team is looking for a tool with support of AWS Bedrock Anthropic Claude (Version 3 and newer).

We found that the current Trulens version (0.33.0) only support Anthropic Claude version 2 and below (with Text Completion API).

Any plan for adding support of Anthropic Claude (Version 3 and newer) to the Trulens? 

Thanks.

**Comment by sfc-gh-jreini** [comment]
Hey @meenonboard - we'll look at adding support for newer Bedrock models soon.

**Comment by sfc-gh-jreini** [comment]
Added https://github.com/truera/trulens/issues/1295 to track

**Reply by meenonboard** [reply]
Appreciate the response. Looking forward to hearing good news.

**Comment by sfc-gh-jreini** [comment]
Hey @meenonboard - support is in this [PR](https://github.com/truera/trulens/pull/1481) and will be in the next release.

Thanks for your patience!

**Reply by meenonboard** [reply]
> Hey @meenonboard - support is in this [PR](https://github.com/truera/trulens/pull/1481) and will be in the next release.
> 
> Thanks for your patience!

Thank you for the update on this change.

**Comment by sfc-gh-nvytla** [comment]
Closing this discussion -- the request has been successfully resolved. Thanks all!
